src/components/PixelCell.jsx

```diff
@@ -25,7 +29,7 @@ export default class PixelCell extends React.Component {
     return (
       <div
         onMouseDown={ev => onMouseDown(id, ev)}
-        onMouseOver={ev => onMouseOver(id, ev)}
+        onMouseOver={ev => hoveredCell(onCellMouseOver(onMouseOver, id, ev, nbrColumns))}
```

Comment:
I'm going to remove this logic since I find it a bit cumbersome. The action will be handled in `onMouseOver`